fragments/filter/README.md

```diff
@@ -762,79 +718,153 @@ This uses the same queryables as Example 4.
 ```
 
-#### Example 6: INTERSECTS cql-json (POST)
+#### Example 7: INTERSECTS cql-json (POST)
 
 ```json
 {
-    "filter": {
-        "intersects": [
-                { "property": "geometry" } ,
-                {
-                   "type": "Polygon",
-                   "coordinates": [[
-                        [-77.0824, 38.7886], [-77.0189, 38.7886],
-                        [-77.0189, 38.8351], [-77.0824, 38.8351],
-                        [-77.0824, 38.7886]
-                    ]]
-                }
+  "filter-lang": "cql-json",
+  "filter": {
+    "intersects": [
+      { "property": "geometry" } ,
+      {
+        "type": "Polygon",
+        "coordinates": [[
+            [-77.0824, 38.7886], [-77.0189, 38.7886],
+            [-77.0189, 38.8351], [-77.0824, 38.8351],
+            [-77.0824, 38.7886]
+        ]]
+      }
+    ]
+  }
+}
+```
+
+### Example 8: Spatial Disjunction
+
+One limitation of the `intersects` parameter is that only a single geometry may be provided. While most
+GeoJSON geometries can be combined to form a composite (e.g., multiple Polygons can be combined to form a
+MultiPolygon), this is much easier to do in the query by combining `INTERSECTS` predicates with the `OR`
+logical operator.
+
+#### Example 8: INTERSECTS cql-text (GET)
+
+```javascript
+/search?filter=INTERSECTS(geometry,POLYGON((-77.0824 38.7886,-77.0189 38.7886,-77.0189 38.8351,-77.0824 38.8351,-77.0824 38.7886))) OR INTERSECTS(geometry,POLYGON((-79.0935 38.7886,-79.0290 38.7886,-79.0290 38.8351,-79.0935 38.8351,-79.0935 38.7886)))
+```
+
+#### Example 8: INTERSECTS cql-json (POST)
+
+```json
+{
+  "filter": { 
+    "or" : [
+      "intersects": [
+        { "property": "geometry" } ,
+        {
+          "type": "Polygon",
+          "coordinates": [[
+            [-77.0824, 38.7886], [-77.0189, 38.7886],
+            [-77.0189, 38.8351], [-77.0824, 38.8351],
+            [-77.0824, 38.7886]
+          ]]
+        }
+      ],
+      "intersects": [
+        { "property": "geometry" } ,
+        {
+          "type": "Polygon",
+          "coordinates": [[
+            [-79.0935, 38.7886], [-79.0290, 38.7886],
+            [-79.0290, 38.8351], [-79.0935, 38.8351],
+            [-79.0935, 38.7886]
+          ]]
+        }
+      ]
+    ]
+  }        
+}
+```
+
+### Example 9: Using IS NULL
+
+One of the main use cases for STAC API is doing cross-collection query. Commonly, this means that items have
+different sets of properties. For example, a collection of Sentinel 2 data may have a property
+`sentinel:data_coverage` and a collection of Landsat 8 data may have a corresponding property
+`landsat:coverage_percent`, both representing what percentage of a given gridded scene actually contains
+data.  However, we many also want to also include in our result items that do not have a value defined for
+either of those properties. 
+
+#### Example 9: cql-text (GET)
+
+```javascript
```

Comment:
Why are these snippets JavaScript? I guess ``http`` is more correct? (you may also want to include GET in front of the call?)

Reply:
cut and paste. I'll change them to have no language.